### DISCUSSION TITLE
Export private conversations, so I have to open them all in the client?

### DISCUSSION THREAD

**Discussion by Feathered-Serpent** [Tyrrrz/DiscordChatExporter#768] [Q&A] [answered]
Hello everyone,

Either I'm not good in searching or it wasn't written with words I searched for.

So if I use DCE it only exports private conversations with people where I have the chat window open in Discord, is that still correct?

So I would need to open a chat with all people I want the messages to be exported? This can be troublesome if having had conversations with people not on the contact list or deleted from the contacts. Difficult to remember all names I ever had contact with.

**Comment by Tyrrrz** [comment, marked as answer]
The other option is to export your data archive in Discord, that should include your no longer opened conversations and the corresponding channel IDs. That would allow you to use those IDs when exporting via the CLI.

**Comment by Feathered-Serpent** [comment]
Export my data archive? Is there a way of exporting data inside Discord as well? Or do you mean the "Request all of my Data" possibility?

**Reply by Tyrrrz** [reply]
> Or do you mean the "Request all of my Data" possibility?

This
